src/test/key_tests.cpp

```diff
@@ -174,18 +174,16 @@ BOOST_AUTO_TEST_CASE(key_signature_tests)
 
     // When entropy is not specified, we should always see low R signatures that are less than 70 bytes in 256 tries
     // We should see at least one signature that is less than 70 bytes.
-    found = true;
     bool found_small = false;
     for (int i = 0; i < 256; ++i) {
         sig.clear();
         std::string msg = "A message to be signed" + ToString(i);
         msg_hash = Hash(msg);
         BOOST_CHECK(key.Sign(msg_hash, sig));
-        found = sig[3] == 0x20;
+        BOOST_CHECK(sig[3] <= 0x20);
```

Comment:
BOOST_CHECK is fairly slow, this is likely why it is not used in the inner loop directly.

Reply:
Do you think it would be better to do

```
if( sig[3] > 0x20 ) {
    found_big = true;
    break;
}
```

and then `BOOST_CHECK(!found_big)` after the loop ?

Also there is `BOOST_CHECK` for `key.Sign()` inside the loop, do you think it should be taken outside the loop, too ?

Reply:
Yes, I think it's better to keep it in a similar way as it is now, assign to a boolean then assert that after the loop.

> Also there is BOOST_CHECK for key.Sign() inside the loop, do you think it should be taken outside the loop, too ?

Probably!  I mean, a loop of 256 isn't that bad, I guess, but there's no reason a tl east to make things worse.